src/platform/windows/events_loop.rs

```diff
@@ -133,6 +141,179 @@ impl Inserter {
     }
 }
 
+#[repr(C)]
+struct FileDropHandlerData {
+    interface: IDropTarget,
+    refcount: AtomicUsize,
+    window: HWND,
+}
+
+struct FileDropHandler {
+    data: *mut FileDropHandlerData
+}
+
+#[allow(non_snake_case)]
+impl FileDropHandler {
+    fn new(window: HWND) -> FileDropHandler {
+        use std::boxed::Box;
+        let data = Box::new(FileDropHandlerData {
+            interface: IDropTarget {
+                lpVtbl: &DROP_TARGET_VTBL as *const IDropTargetVtbl
+            },
+            refcount: AtomicUsize::new(1),
+            window
+        });
+        FileDropHandler {
+            data: Box::into_raw(data)
+        }
+    }
+
+    // Implement IUnknown
+    pub unsafe extern "system" fn QueryInterface(
+        _this: *mut unknwnbase::IUnknown,
+        _riid: REFIID,
+        _ppvObject: *mut *mut c_void,
+    ) -> HRESULT {
+        unimplemented!();
```

Comment:
The reason I left it unimplemented is that I couldn't find a way to implement it in Rust. It seems that this function is not required to be implemented for an `IDropTarget` but if anyone knows how the implementation should look, it would certainly be nice to have. In the meantime I'll just add a comment with the following text: "This function doesn't appear to be required for an IDropTarget.
An implementation would be nice however."

Reply:
Sounds good!

Reply:
(sorry about the redundant messages; I was having network problems yesterday)